chia/full_node/block_store.py

```diff
@@ -350,6 +357,36 @@ async def get_block_info(self, header_hash: bytes32) -> Optional[GeneratorBlockI
                     b.foliage.prev_block_hash, b.transactions_generator, b.transactions_generator_ref_list
                 )
 
+    async def get_plot_filter_info(self, header_hash: bytes32) -> Optional[PlotFilterInfo]:
```

Comment:
I really think it's a mistake to add this function. The block store is a serious bottleneck in the full node, please avoid more round-trips to the database. In the places where you need these fields out of the full block, add it to the existing query.